### PR TITLE
fixes #6067 - UI inconsistency around root password limit 8 characters

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/models/hostgroup.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ class Hostgroup < ActiveRecord::Base
   has_many :user_hostgroups, :dependent => :destroy
   has_many :users, :through => :user_hostgroups
   validates :name, :format => { :with => /\A(\S+\s?)+\Z/, :message => N_("can't be blank or contain trailing white spaces.")}
+  validates :root_pass, :allow_blank => true, :length => {:minimum => 8, :message => _('should be 8 characters or more')}
```
Comment (ohadlevy) [inline app/models/hostgroup.rb:15]: move to host_common?

Comment (isratrade) [inline app/models/hostgroup.rb:15]: @ohadlevy, managed.rb has `presence => {:message => N_('should not be blank - consider setting a global or host group default')}` whereas hostgroup.rb has `allow_blank => true`
The `:length` validation part can be in host_common

**Comment by isratrade** [comment]
@ohadlevy, manged.rb has `:if => Proc.new { |host| host.managed && pxe_build?`

File: app/models/hostgroup.rb (hunk 1 above)
Comment (ohadlevy) [inline app/models/hostgroup.rb:15]: OK :)

**Comment by ohadlevy** [comment]
:+1: 
should we add a test?

**Comment by thomasmckay** [comment]
Any chance of removing 8 char validation completely?

**Comment by domcleal** [comment]
One day it'll be configurable: http://projects.theforeman.org/issues/5663

**Comment by domcleal** [comment]
Merged as 9c30b63 for Foreman 1.6.0, thanks @isratrade.

**Pull request closed without merging**